components/calendar/calendar.js

```diff
@@ -630,6 +630,12 @@ class Calendar extends LocalizeCoreElement(RtlMixin(LitElement)) {
 		await this._updateFocusDate(date, false, allowDisabled);
 	}
 
+	showYearAndMonth(year, month) {
+		if (year === undefined || month === undefined) return;
+		this._shownYear = year;
+		this._shownMonth = month;
```

Comment:
Yeah, the `Date` object provides a 0-based month. I could be convinced to accept a 1-based month here. I mostly just stuck with the 0-based to be consistent, but I'm not sure which way is the least confusing... inconsistency or 0-based. Thoughts?

Reply:
Hmm that's tricky. I think it's probably better to be consistent and do the 0-based, but also add a comment and/or some documentation around this.

Reply:
Added to docs.

An alternative would be to accept a `Date` object. Could also name this `showDate` instead, but that's a little ambiguous because the `Date` they provide could be in the previous or next month and already in view.

Reply:
They could go `showYearAndMonth(new Date(2020, 3))` which is April 1st, 2020. It would still be 0-based, but then it's the browser imposing that weirdness instead of us. 🤷‍♂️

Reply:
I'm good either way but I think it would be fine to leave it as is - looking at the code, there are a lot of functions that take the 0-based month and year as arguments so it would be consistent there.

Reply:
I think I like tightening up the arg to be a `Date`, so I updated.